book/07-git-tools/sections/submodules.asc

```diff
@@ -682,7 +682,9 @@ CONFLICT (submodule): Merge conflict in DbConnector
 Automatic merge failed; fix conflicts and then commit the result.
 ----
 
-What it's suggesting that you do is to update the index like you had run `git add`, which clears the conflict, then commit. You probably shouldn't do this though. You can just as easily go into the submodule directory, see what the difference is, fast-forward to this commit, test it properly, and then commit it.
+What it's suggesting that you do is to update the index like you had run `git add`, which clears the conflict, then commit.
```

Comment:
This line is a bit clunky to read. But I myself have no clear improvement. Maybe @ben can offer help with a light rewrite of this line?
>What it's suggesting that you do is to update the index like you had run `git add`, which clears the conflict, then commit.

Reply:
Sorry for the delay. How about:

> The suggested command Git is providing will update the index as though you had run `git add` (which clears the conflict), then commit.

Reply:
Hi @ben, the new line is a lot better.

---

Hi @aollier do you want to fix the line? Or would you like me to make a new pull-request?

Reply:
Ok I do it right now.